src/test/java/org/apache/commons/imaging/ImageDumpTest.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.commons.imaging;
+
+import org.junit.Test;
+
+import java.awt.color.ColorSpace;
+import java.awt.image.BufferedImage;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit tests for class {@link ImageDump}.
+ *
+ * @date 2017-07-13
+ * @see ImageDump
+ *
+ **/
+public class ImageDumpTest{
+
+
+    @Test
+    public void testDumpColorSpace() {
+
+        ImageDump imageDump = new ImageDump();
+        ColorSpace colorSpace = ColorSpace.getInstance(1004);
+        imageDump.dumpColorSpace("Ku&]N>!4'C#Jzn+", colorSpace);
+
+        assertEquals(3, colorSpace.getNumComponents());
+
+    }
```

Comment:
Most test methods I've seen in commons-imaging prefer to have no blank lines at the start and end of the test method body. I suggest removing the blank lines at the start and end of `testDumpColorSpace`'s body here for consistency, so like this:

```java
@Test 
public void testDumpColorSpace() {
ImageDump imageDump = new ImageDump();
        ColorSpace colorSpace = ColorSpace.getInstance(1004);
        imageDump.dumpColorSpace("Ku&]N>!4'C#Jzn+", colorSpace);

        assertEquals(3, colorSpace.getNumComponents());
}